plugins/SimulationView/SimulationViewMenuComponent.qml

```diff
@@ -133,7 +132,54 @@ Cura.ExpandableComponent
             width: parent.width
             model: layerViewTypes
             visible: !UM.SimulationView.compatibilityMode
-            style: UM.Theme.styles.combobox
```

Comment:
perhaps this should also be combined with the work done for CURA-8378

Reply:
Have already been talking to @Joeydelarago  how we could possible combine our work. We identified 3 possible course of actions to upgrade the checkboxes.

1. Change all checkboxes in this commit, however this would result in a lot of merge conflicts and possibly duplicate work.
2. Make a PR to the CURA-8378 branch with the upgraded checkboxes, which makes things way too complicated.
3. Wait untill CURA-8378 is merged, and then do the proper upgrade, which is the course of action we choose.